gpytorch/lazy/lazy_tensor.py

```diff
@@ -1800,6 +1859,30 @@ def __getitem__(self, index):
         # We're done!
         return res
 
+    @cached(name="svd")
+    def _svd(self) -> Tuple["LazyTensor", Tensor, "LazyTensor"]:
+        """Method that allows implementing special-cased SVD computation. Should not be called directly"""
+        from gpytorch.lazy.non_lazy_tensor import NonLazyTensor
+
+        U, S, V = torch.svd(self.evaluate())
+        return NonLazyTensor(U), S, NonLazyTensor(V)
```

Comment:
Following some testing, using symeig throughout is preferable:

```suggestion
        U, S = self.symeig(eigenvectors=True)
        V = U.transpose(-1, -2)

        return U, S, V
```